core/interfaces.go

```diff
@@ -135,4 +135,5 @@ type StorageAuthority interface {
 // Publisher defines the public interface for the Boulder Publisher
 type Publisher interface {
 	SubmitToCT(ctx context.Context, der []byte) error
+	SubmitToSingleCT(ctx context.Context, logID string, der []byte) error
```

Comment:
Filed https://github.com/letsencrypt/boulder/issues/2357 for changing the existing `SubmitToCT` API.